pkg/controller/pravegacluster/upgrade_test.go

```diff
@@ -221,6 +225,51 @@ var _ = Describe("Pravega Cluster Version Sync", func() {
 					Ω(upgradeCondition.Message).Should(Equal("0"))
 				})
 			})
+			Context("Upgrade Segmentstore to empty version", func() {
+				var (
+					foundPravega *v1beta1.PravegaCluster
+				)
+				BeforeEach(func() {
+					foundPravega = &v1beta1.PravegaCluster{}
+					_ = client.Get(context.TODO(), req.NamespacedName, foundPravega)
+					foundPravega.Status.TargetVersion = ""
+
+					foundPravega.Status.SetPodsReadyConditionTrue()
+					client.Update(context.TODO(), foundPravega)
+					_, _ = r.Reconcile(req)
+					foundPravega = &v1beta1.PravegaCluster{}
+					_ = client.Get(context.TODO(), req.NamespacedName, foundPravega)
+
+				})
+
+				It("should set upgrade condition reason and message to empty", func() {
+					_, upgradeCondition := foundPravega.Status.GetClusterCondition(pravegav1beta1.ClusterConditionUpgrading)
```

Comment:
Can we check that UpgradeCondition is set to false as well, since `clearUpgradeStatus` would do that

Reply:
done